.projenrc.js

```diff
@@ -1,7 +1,6 @@
 const { awscdk } = require('projen');
 const project = new awscdk.AwsCdkTypeScriptApp({
   cdkVersion: '2.22.0',
-  cdkVersionPinning: true,
```

Comment:
zodat projen de CDK ook update (maar niet naar 3.x)

Reply:
en waarom nu niet meteen naar 2.25 (de laatste)?

Reply:
Dat is wat verwarrend: 
Dat doet ie wel, maar eigenlijk staat hier nu: cdkVersion: "^2.22.0" (alles tussen 3.0.0 en 2.22.0 exclusief 3.0.0), door pinning uit te zetten.